website/docs/r/sourcerepos_repository.html.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: "google"
+page_title: "Google: google_sourcerepos_repository"
+sidebar_current: "docs-google-sourcerepos_repository"
+description: |-
+  Manages repositories within Google Cloud Source Repositories.
+---
+
+# google\_sourcerepos\_repository
+
+Manages repositories within Google Cloud Source Repositories.
```

Comment:
Can you add links to the Google Cloud documentation. I suggest appending the following to this line:
```markdown
For more information, see [the official
documentation](https://cloud.google.com/compute/docs/source-repositories) and
[API](https://cloud.google.com/source-repositories/docs/reference/rest/v1/projects.repos)
```